src/api-handler.ts

```diff
@@ -12,10 +12,24 @@
 
 'use strict';
 
+import { AddonManagerProxy } from "./addon-manager-proxy";
+
+export interface APIRequestOptions {
```

Comment:
```suggestion
export interface APIRequestOptions {
  method: string
  path: string
  query: {}
  body: {}
}
```
based on constructor.